salt/modules/network.py

```diff
@@ -1466,6 +1598,39 @@ def get_route(ip):
 
         return ret
 
+    if __grains__['kernel'] == 'AIX':
+        # root@la68pp002_pub:~# route -n get 172.29.149.95
+        #   route to: 172.29.149.95
+        #destination: 172.29.149.95
+        #    gateway: 127.0.0.1
+        #  interface: lo0
+        #interf addr: 127.0.0.1
+        #     flags: <UP,GATEWAY,HOST,DONE,STATIC>
+        #recvpipe  sendpipe  ssthresh  rtt,msec    rttvar  hopcount      mtu     expire
+        #      0         0         0         0         0         0         0    -68642
+        cmd = 'route -n get {0}'.format(ip)
+        out = __salt__['cmd.run'](cmd, python_shell=False)
+
+        ret = {
+            'destination': ip,
+            'gateway': None,
+            'interface': None,
+            'source': None
+        }
+
+        for line in out.splitlines():
+            line = line.split(':')
+            if 'route to' in line[0]:
```

Comment:
On AIX if the route does not exist, it still outputs the commented out guide at line# 1602,

root@la68pp002_pub:/srv/pillar# route -n get 192.168.0.1   
   route to: 192.168.0.1
destination: default
       mask: default
    gateway: 172.29.128.13
  interface: en0
interf addr: 172.29.149.95
      flags: <UP,GATEWAY,DONE>
 recvpipe  sendpipe  ssthresh  rtt,msec    rttvar  hopcount      mtu     expire
       0         0         0         0         0         0         0   -315505 

If bad values are input then 
root@la68pp002_pub:/srv/pillar# route -n get 123.321.234.234
0821-287 123.321.234.234: bad value

And the pre-populated ret is returned, similar to the code for the other platforms, e.g.. OpenBSD, Solaris, etc.